apps/dapp/src/hooks/api/use-react-query.ts

```diff
@@ -0,0 +1,40 @@
+import { UseQueryResult, useQuery } from '@tanstack/react-query';
+
+// Centralize all the dApp react query keys in case we need to cancel or invalidate them
+// through the app, this makes it easier to track them, please add new ones as required
+export const getQueryKey = {
+  txPagDefault: () => ['getTxPaginationDefaultValues'],
+  txHistory: () => ['getTxHistory'],
+  metrics: (s?: StrategyKey) => {return s ? ['getMetrics', s] : ['getMetrics']},
+  trvMetrics: () => ['getTreasureReserveMetrics'],
+}
+
+export enum StrategyKey {
```

Comment:
will set it back to the original file for now, then we can create the v2 types file

Reply:
I've returned the `StrategyKey` to the original file, and added a new enum `ALL`, because I'm also using it for my table component, but if that's an issue lmk and I'll create a new one